paddle_billing/Entities/Shared/PaymentMethodType.py

```diff
@@ -9,6 +9,6 @@ class PaymentMethodType(PaddleStrEnum, metaclass=PaddleStrEnumMeta):
     GooglePay: "PaymentMethodType" = "google_pay"
     Ideal: "PaymentMethodType" = "ideal"
     Offline: "PaymentMethodType" = "offline"
-    Paypal: "PaymentMethodType" = "paypal"
+    PayPal: "PaymentMethodType" = "paypal"
```

Comment:
@davidgrayston-paddle this might be a breaking change, let's keep what we originally had